### DISCUSSION TITLE
Up and down arrow key not working

### DISCUSSION THREAD

**Discussion by thomasdevine01** [marlonrichert/zsh-autocomplete#723] [Q&A]
On install, the up and down arrow keys do not work. I was able to solve this issue by running source, and then the location of the repo and then running it again in my ~/.zshrc file. I am currently using the version supplied on the repo, and I am also using a mac m2 pro. 

Suggest potentially adding a note in the install guide, or a note somewhere in the readme.

**Comment by silijon** [comment]
I also hit this issue.

Turns out sourcing oh-my-zsh.sh -after- sourcing zsh-autocomplete breaks the up and down arrow keys. Solution was to source oh-my-zsh.sh further up in .zshrc.

I agree with @thomasdevine01 that it may be worth a note. Was pretty annoying before figuring that out.

**Comment by marlonrichert** [comment]
There are now some [notes about this in the Readme](https://github.com/marlonrichert/zsh-autocomplete/blob/main/README.md#caveats).

Quote of the relevant text:
> * Plugins or other scripts that you load _after_ loading Autocomplete may override these bindings.
>   If you find that some shortcuts don't work as expected, then you can fix them by
>   * changing the order in which you source your plugins or by
>   * running [`bindkey` commands](#reassign-keys) in your dotfiles _after_ you source your plugins.
ey` commands](#reassign-keys) in your dotfiles _after_ you source your plugins.
